test/recipes/80-test_ssl_old.t

```diff
@@ -822,4 +822,37 @@ sub testssl {
         ok(run(test([@ssltest, "-bio_pair", "-tls1", "-ct"])));
         }
     };
+
+    subtest 'Session ticket + SNI enable tests' => sub {
+	######################################################################
```

Comment:
ssltest_old.c is deprecated and new tests should be added to the new test framework. Please read test/README.ssltest.md and ping me if anything is unclear.

Otherwise it's an uphill battle for me to ever get rid of the old test.